client/scss/utilities/_root-scope-classes.scss

```diff
@@ -273,11 +273,8 @@
   border-radius: $border-radius-unit;
 }
 
-.rounded-top-left {
+.rounded-diagonal {
```

Comment:
I'll start a sweepstake on when we're going to need top-right/bottom-left rounded corners!

Reply:
But let's ask the question why then.

The reason the other abstractions have come about is because we've used them in a million places, these have only ever been used in this way.

Happy to go with your feel on this though as you're closer to it.

Reply:
(I'm cool with this – only kidding, really)